cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -367,14 +381,48 @@ def _generate_metric_filter_pattern(event_type, failure_type=None):
                 ),
             ]
 
-            cluster_common_errors.append(
-                _ErrorMetric(
+            cluster_health_metrics.append(
+                _HealthMetric(
                     "Custom Action Errors",
                     custom_action_errors,
+                    left_y_axis=cloudwatch.YAxisProps(min=0.0),
```

Comment:
Question: why do we need to set left_y_axis here?

Reply:
So that we can guarantee that the y-origin is always 0 seconds.